turbinia/client.py

```diff
@@ -95,6 +95,24 @@
 logger.setup()
 
 
+def get_turbinia_client(run_local=False):
+  """Return Turbinia client based on config.
+
+  Returns:
+    Initialized BaseTurbiniaClient or TurbiniaCeleryClient object.
+  """
+  config.LoadConfig()
+  # pylint: disable=no-else-return
+  if config.STATE_MANAGER.lower() == 'datastore':
+    return BaseTurbiniaClient(run_local=run_local)
+  elif config.STATE_MANAGER.lower() == 'redis':
+    return TurbiniaCeleryClient()
```

Comment:
I think you can actually pass in run_local here as well.  I'm not exactly sure why it wasn't passed into the client invocations before -- I think possibly because it used to determine whether it was local vs. gcp by using flags, so the caller would just not specify the celery flag, but now that it comes from the config alone, we should probably add it here too.